qcfractal/interface/collections/collection.py

```diff
@@ -57,12 +57,6 @@ class DataModel(BaseModel):
         provenance: Dict[str, str] = {}
         id: str = 'local'
 
-        class Config:
```

Comment:
Did you want datamodel to be mutable?

Reply:
This is never called so its attributes do not matter as Config is no inherited. But in general, yes the datamodel should be mutable.